quickref/qsharp-quick-reference.tex

```diff
@@ -342,6 +609,73 @@
+	
+	\usebox\matrixscalarleft &\usebox\matrixscalarright \\
+	
+	\usebox\matrixmultiplicationleft & \usebox\matrixmultiplicationright \\
+
+	\usebox\matrixtransposeleft & \usebox\matrixtransposeright \\	 
+	
+	\usebox\matrixadjointleft & \usebox\matrixadjointright \\
+
+	\usebox\innerproductleft & \usebox\innerproductright \\ 	
+	
+	\usebox\outerproductleft & \usebox\outerproductright \\
+	
+\end{keysref}
+
+\newpage
+
+\section{Gates reference}
+
+
+\begin{keysrefQbits}{Single Qubit gates}
+
+	Gate & Matrix representation    & Ket-Bra & Applying to \( |\psi\rangle = \alpha|0\rangle + \beta|1\rangle \) & Applying to basis states & & \\
+	
+	 X & \usebox\xgate & \( |0\rangle\langle1| + |1\rangle\langle0|\) & \( |\psi\rangle = \alpha|1\rangle + \beta|0\rangle \) & \( X|0\rangle = |1\rangle \newline X|1\rangle = |0\rangle \) & \(
+		 X|+\rangle = |+\rangle \newline   X|-\rangle = -|-\rangle  \) & \( X|i\rangle = i|-i\rangle \newline X|-i\rangle = -i|i\rangle \)  	\\
+	Y & \usebox\ygate & \( i(|1\rangle\langle0| - |0\rangle\langle1|)\) & \(Y|\psi\rangle = i\big(\alpha|1\rangle - \beta|0\rangle\big)\) & \(Y|0\rangle = i|1\rangle \newline Y|1\rangle = -i|0\rangle \) & \( Y|+\rangle = -i|-\rangle  \newline
+		 Y|-\rangle = i|+\rangle \) & \( Y|i\rangle = |i\rangle \newline Y|-i\rangle = -|-i\rangle\)   \\
+	Z & \usebox\zgate & \( |0\rangle\langle0| - |1\rangle\langle1| \) & \( Z|\psi\rangle = \alpha|0\rangle - \beta|1\rangle \) & \( Z|0\rangle = |0\rangle \newline Z|1\rangle = -|1\rangle \) & \(  Z|+\rangle = |-\rangle \newline
+		 Z|-\rangle = |+\rangle \) & \( Z|i\rangle = |-i\rangle \newline Z|-i\rangle = |i\rangle \)  \\
+	I & \usebox\igate & \( |0\rangle\langle0| + |1\rangle\langle1| \)  & \( I|\psi\rangle = |\psi\rangle \) & & & \\
+	H & \usebox\hgate & \( |0\rangle\langle+| + |1\rangle\langle-| \) & \( H|\psi\rangle = \alpha|+\rangle + \beta|-\rangle = \newline \frac{\alpha + \beta}{\sqrt{2}}|0\rangle + \frac{\alpha - \beta}{\sqrt{2}}|1\rangle \) & 
+		\( H|0\rangle = |+\rangle 	\newline H|1\rangle = |-\rangle \) & \( H|+\rangle = |0\rangle \newline H|-\rangle = |1\rangle \) & \( H|i\rangle = e^{i\pi/4}|-i\rangle \newline H|-i\rangle = e^{-i\pi/4}|i\rangle \) \\
+	S & \usebox\sgate & \( |0\rangle\langle0| + i|1\rangle\langle1| \) & \( S|\psi\rangle = \alpha|0\rangle + i\beta|1\rangle \) & \( S|0\rangle = |0\rangle \newline S|1\rangle = i|1\rangle \) & \( S|+\rangle = |i\rangle 
+		\newline S|-\rangle = |-i\rangle \) & \( S|i\rangle = |-\rangle \newline S|-i\rangle = |+\rangle \)  \\
+	T & \usebox\tgate & \(|0\rangle\langle0| + e^{i\pi/4}|1\rangle$$\langle1| \) & \( T|\psi\rangle = \alpha|0\rangle + e^{i\pi/4} \beta |1\rangle \) & \( T|0\rangle = |0\rangle \newline T|1\rangle = e^{i\pi/4}|1\rangle \) & &  \\
+	\( R_x(\theta) \) & \usebox\rxgate & & \( R_x(\theta)|\psi\rangle = \ (\alpha\cos\frac{\theta}{2} - i\beta\sin\frac{\theta}{2})|0\rangle + (\beta\cos\frac{\theta}{2} - i\alpha\sin\frac{\theta}{2})|1\rangle \) & 
```

Comment:
I'd add ket-bra representation to rotation hates as well - it will be quite painful for Rx and Ry but the other two are easy, and this will give the table a nice complete look